pymc3/distributions/continuous.py

```diff
@@ -924,6 +925,73 @@ def logcdf(self, value):
         )
 
 
+class ZeroSumNormal(Continuous):
+    def __new__(cls, name, *args, **kwargs):
+        zerosum_axes = kwargs.get("zerosum_axes", None)
+        zerosum_dims = kwargs.get("zerosum_dims", None)
+        dims = kwargs.get("dims", None)
+
+        if isinstance(zerosum_dims, str):
+            zerosum_dims = (zerosum_dims,)
+        if isinstance(dims, str):
+            dims = (dims,)
+
+        if zerosum_dims is not None:
+            if dims is None:
+                raise ValueError("zerosum_dims can only be used with the dims kwargs.")
+            if zerosum_axes is not None:
+                raise ValueError("Only one of zerosum_axes and zerosum_dims can be specified.")
+            zerosum_axes = []
+            for dim in zerosum_dims:
+                zerosum_axes.append(dims.index(dim))
+            kwargs["zerosum_axes"] = zerosum_axes
+
+        return super().__new__(cls, name, *args, **kwargs)
+
+    def __init__(self, sigma=1, zerosum_axes=None, zerosum_dims=None, **kwargs):
```

Comment:
Note that `zerosum_dims` is not used in `__init__`, but if I don't put it here, it doesn't seem to be passed on to `__new__`: `TypeError: __init__() got an unexpected keyword argument 'zerosum_dims'`
Not sure we can do it otherwise though. If someone has a better idea, I'm all ears